website/events/models/event_registration.py

```diff
@@ -151,7 +151,7 @@ def payment_notes(self):
 
     @property
     def payment_payer(self):
-        return self.member
+        return PaymentUser.objects.get(pk=self.member.pk) if self.member else None
```

Comment:
Why does this need to be a PaymentUser exactly? The place where this property is used could convert it, then you don't need to import the model here in events.